### DISCUSSION TITLE
New maintainer!

### DISCUSSION THREAD

**Discussion by fmartingr** [go-shiori/shiori#326] [Announcements]
Hello there! I'm @fmartingr and I've volunteered to be a maintainer of Shiori along with the current team. I'm working as a Go engineer by profession, love my job but wanted to contribute to Open Source in a better way.

The project has been hibernating while work has been piling up, and it's a shame! I was one of the people that forked the project and wanted to make it's own "better and improved" Shiori.

Problem with that is that a lot of people already fell in love with the original, dedicating their time into it for either small or big improvements or commenting on the issues helping the community. I came into reason and contacted the team (#256) in order to help here rather than following the traditional ["14 standards" approach](https://xkcd.com/927/).

I just wanted to give everyone a little presentation and some hope for the project, later this week I will post a little road map of the things pending and how I'm planning to move forward.

Thanks all,
F.

**Comment by symgryph** [comment]
Welcome! Its nice to see a new maintainer. I can help with docs/security issues.

**Comment by symgryph** [comment]
Oh and what's the channel on irc chat?

**Comment by symgryph** [comment]
NM! Found it in commit: https://web.libera.chat/#shiori

**Comment by amitjethani** [comment]
Really appreciate you taking over @fmartingr 
Having a donation or other link (on github, ko-fi, etc.) to support the work would be swell 👍

**Comment by yaneony** [comment]
Is this project still maintained or is it dead again?

**Reply by fmartingr** [reply]
I'm still here, just a bit scarce on time lately. I try to review everything every once in a while so PRs don't stale, but have little time to do some actual coding myself.

**Reply by yaneony** [reply]
Good to know! Thanks for your work!

**Comment by d3vw** [comment]
Appreciated!
